src/assets/scss/_input-switch.scss

```diff
@@ -7,6 +7,14 @@
     height: auto;
   }
 
+  .has-scrollbar {
+    padding-right: 2rem !important;
```

Comment:
Add a comment above this rule saying that you're adding this rule to increase the space to accommodate for scrollbar